app/utilities/decimal_places.py

```diff
@@ -92,14 +92,16 @@ def custom_format_unit(
         value=value,
         measurement_unit=measurement_unit,
         length=length,
-        format=number_format,
+        format=number_format,  # type: ignore
```

Comment:
Above this it would be good to have a `Type ignore: babel function has incorrect typing, NumberPattern is valid` or something to that effect for future reference